tests/baselines/reference/jsdocPropertyTagInvalid.errors.txt

```diff
@@ -1,12 +1,13 @@
-/a.js(3,15): error TS2304: Cannot find name 'sting'.
+/a.js(3,15): error TS2552: Cannot find name 'sting'. Did you mean 'String'?

```

Comment:
hmm, this also needs to suggest `string` not `String`. We might need to improve the suggestion code, perhaps with a special case for string/number/boolean/object (and any others?).

Reply:
Looks like `string/number/boolean/object/bigint` etc are missing from the list of global suggestions.